quadhoral-openmeteo/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM public.ecr.aws/lambda/python:latest
```

Comment:
A goods functional Dockerfile for the creation of the ECR image for the weather pipeline. Importantly, no .env file is needed (as it is unsafe), because all necessary environment variables are injected in the Lambda at run time because of the terraform configuration.